Update MS DNS README (#33053)
demisto/content: Packs/MicrosoftDNS/README.md

When configuring the instance, you should use a yml file that configures the vendor and product, as shown in the below configuration for the Microsoft DNS product.

**Pay Attention**:

> * When using this pack there are two integrations available for it.
> * Timestamp log ingestion is supported for either of the formats- **%m/%d/%Y %I:%M:%S %p** OR **%d/%m/%Y %H:%M:%S** in UTC (00:00) time.

Reviewer: ```suggestion
* There are two integrations available in this content pack.
* Timestamp log ingestion is supported in either of the following formats in UTC (00:00) time.
   - *%m/%d/%Y %I:%M:%S %p*
   - *%d/%m/%Y %H:%M:%S*
```
